relay-event-derive/src/lib.rs

```diff
@@ -332,7 +321,8 @@ struct FieldAttrs {
     characters: Option<TokenStream>,
     max_chars: Option<TokenStream>,
     max_chars_allowance: Option<TokenStream>,
-    bag_size: Option<TokenStream>,
+    max_struct_depth: Option<TokenStream>,
+    max_struct_bytes: Option<TokenStream>,
```

Comment:
Can we call these `max_depth` and `max_bytes`? IMO the `_struct_` infix does not really provide information.